src/web/client/extension.ts

```diff
@@ -460,3 +463,28 @@ function isActiveDocument(fileFsPath: string): boolean {
         WebExtensionContext.fileDataMap.getFileMap.has(fileFsPath)
     );
 }
+
+async function logArtemisTelemetry() {
+
+    try {
+        const orgId = WebExtensionContext.urlParametersMap.get(
+            queryParameters.ORG_ID
+        ) as string
+
+                const artemisResponse = await fetchArtemisResponse(orgId, WebExtensionContext.telemetry.getTelemetryReporter());
+
+        if (!artemisResponse) {
+            return;
+        }
+
+        const { geoName } = artemisResponse[0];
+        WebExtensionContext.telemetry.sendInfoTelemetry(telemetryEventNames.WEB_EXTENSION_ARTEMIS_RESPONSE,
+            { orgId: orgId, geoName: String(geoName) });
+    } catch (error) {
+        WebExtensionContext.telemetry.sendErrorTelemetry(
+            telemetryEventNames.WEB_EXTENSION_ARTEMIS_RESPONSE_FAILED,
+            logArtemisTelemetry.name,
+            (error as Error)?.message,
```

Comment:
Please don't use telemetry as replacement for logging

Reply:
This code runs in VSCode Web. Here we don't have the option to log in PAC Log file.